cfme/infrastructure/config_management.py

```diff
@@ -520,3 +431,103 @@ def __init__(self, name=None, url=None, ssl=None, credentials=None, key=None):
         self.ssl = ssl
         self.credentials = credentials
         self.key = key or name
+
+
+@navigator.register(ConfigManager, 'All')
+class MgrAll(CFMENavigateStep):
```

Comment:
Why MgrAll and SysAll instead of 'All', and further why not MgrAdd if MgrAll is necessary?